--- modules/prompt/external/pure/pure.zsh	2018-10-27 13:26:23.000000000 +0900
+++ /Users/nakayama/prezto/modules/prompt/external/pure/pure.zsh	2018-10-27 13:06:48.000000000 +0900
@@ -115,7 +115,7 @@ prompt_pure_preprompt_render() {
 	local -a preprompt_parts
 
 	# Set the path.
-	preprompt_parts+=('%F{cyan}%~%f')
+	preprompt_parts+=('%F{blue}%~%f')
 
 	# Add git branch and dirty status info.
 	typeset -gA prompt_pure_vcs_info
@@ -529,7 +529,7 @@ prompt_pure_state_setup() {
 	fi
 
 	# show username@host if logged in through SSH
-	[[ -n $ssh_connection ]] && username='%F{242}%M%F'
+	username='%F{242}%n@%m%f'
 
 	# show username@host if root, with username in white
 	[[ $UID -eq 0 ]] && username='%F{white}%n%f%F{242}@%m%f'
@@ -584,7 +584,7 @@ prompt_pure_setup() {
 	PROMPT='%(12V.%F{242}%12v%f .)'
 
 	# prompt turns red if the previous command didn't exit with 0
-	PROMPT+='%(?.%F{cyan}.%F{green})${prompt_pure_state[prompt]}%f '
+	PROMPT+='%(?.%F{magenta}.%F{red})${prompt_pure_state[prompt]}%f '
 
 	# Store prompt expansion symbols for in-place expansion via (%). For
 	# some reason it does not work without storing them in a variable first.
