--- prezto/runcoms/zpreztorc	2018-10-27 23:45:33.939070771 +0900
+++ /home/tsnakaya/.zprezto/runcoms/zpreztorc	2018-10-27 23:22:02.254367188 +0900
@@ -35,6 +35,8 @@
   'spectrum' \
   'utility' \
   'completion' \
+  'syntax-highlighting' \
+  'autosuggestions' \
   'prompt'

 #
@@ -113,7 +115,7 @@
 # Set the prompt theme to load.
 # Setting it to 'random' loads a random theme.
 # Auto set to 'off' on dumb terminals.
-zstyle ':prezto:module:prompt' theme 'sorin'
+zstyle ':prezto:module:prompt' theme 'pure'

 # Set the working directory prompt display length.
 # By default, it is set to 'short'. Set it to 'long' (without '~' expansion)
